test/aqua/test_grover.py

```diff
@@ -54,7 +54,7 @@ def test_grover(self, input_test, sol, oracle_cls, mct_mode, simulator, optimiza
         """ grover test """
         groundtruth = sol
         oracle = oracle_cls(input_test, optimization=optimization)
-        grover = Grover(oracle, incremental=True, mct_mode=mct_mode)
+        grover = Grover(oracle, incremental=True, rotation_counts=[int(1.34 ** i) for i in range(20)], mct_mode=mct_mode)
```

Comment:
Here you're specifying explicitly what's also used as default value. 
Could you instead add a second test where you explicitly change the rotation counts to something else than the default?

Reply:
Thank you for the comment, please see the current solutions.